tests_common/test_utils/db.py

```diff
@@ -84,7 +83,15 @@ def initial_db_init():
     flask_app = Flask(__name__)
     flask_app.config["SQLALCHEMY_DATABASE_URI"] = conf.get("database", "SQL_ALCHEMY_CONN")
     init_appbuilder(flask_app)
-    get_auth_manager().init()
+
+    if AIRFLOW_V_3_0_PLUS:
+        from airflow.api_fastapi.app import get_auth_manager
+
+        get_auth_manager().init()
+    else:
+        from airflow.www.extensions.init_auth_manager import get_auth_manager
+
+        get_auth_manager().init()
```

Comment:
```suggestion
    else:
        from airflow.www.extensions.init_auth_manager import get_auth_manager

    get_auth_manager().init()
```

nit

Reply:
Oops, I just saw that one. It is better I agree